Server/src/main/java/local/AssetDAO.java

```diff
@@ -5,18 +5,17 @@
   @version 1.0
   @last_edit 12/27/2020
  */
-package com.cbms.source.local;
+package local;
 
-import com.cbms.app.TrainedModel;
-import com.cbms.app.item.Asset;
-import com.cbms.app.item.AssetType;
+import app.TrainedModel;
+import app.item.Asset;
 
 import java.util.ArrayList;
 
 public interface AssetDAO {
     ArrayList<Asset> getAssetsToUpdate();
 
-    Asset getAssetToUpdate(int assetID);
+    //Asset getAssetToUpdate(int assetID);
```

Comment:
Is this still useful as well? Dead code should always be removed

Reply:
Code cleanup is out of scope of this pr